### ISSUE TITLE
EF Core and 4.5.1 support

### ISSUE THREAD

**Issue by devtriad** [dotnet/efcore#6332]
I have an aspnet core  application

I have an ASP.NET Core web app that targets 4.51 framework

When it targets 451 I get the error

SqlException: Invalid object name 'AspNetUsers'.

When I don't target it I don't get the error and my application works.

In my OnModelCreating method 

```
        protected override void OnModelCreating(ModelBuilder builder)
        {

            base.OnModelCreating(builder);
            //rename table names
            builder.Entity<ApplicationUser>().ToTable("tbl_Users");
            builder.Entity<ApplicationRole>().ToTable("tbl_Roles");
            builder.Entity<IdentityUserRole<Guid>>().ToTable("tbl_UserRoles");
            builder.Entity<IdentityUserClaim<Guid>>().ToTable("tbl_UserClaims");
            builder.Entity<IdentityUserLogin<Guid>>().ToTable("tbl_UserLogins");
            builder.Entity<IdentityRoleClaim<Guid>>().ToTable("tbl_RoleClaims");
            builder.Entity<IdentityUserToken<Guid>>().ToTable("tbl_UserTokens");
```

In project.json

```
  "frameworks": {
     "net452": {
      "frameworkAssemblies": {
         "WindowsBase": "4.0.0"
        }
     }
  },
```

Does entity framework core support 451?

**Comment by devtriad** [comment]
I upgraded to 4.5.2 so feel free to close it since 4.5.1 is no longer supported by MS.

**Comment by divega** [comment]
Reopening to try to repro since this is still unexpected behavior.

**Comment by divega** [comment]
I tried this with 4.5.1 but could not repro. I customized the Identity model to have `ApplicationUser` derive from `IdentityUser<Guid>`, added a similar `ApplicationRole` and made the correspoinding adjustments to the call to `AddIdentity()` in `Startup`. With the snippet of code that renames the tables in `OnModelCreating()` of the application context all the tables were named as requested and there was no table name `AspNetUsers` left in the model.
